src/app/designationEditor/personalOptionsModal/personalOptionsModal.tpl.html

```diff
@@ -65,17 +71,30 @@
           </tfoot>
         </table>
       </div>
-
+      <div class="tab-pane"
+           ng-class="{active: $ctrl.tab === 'newsletter'}"
+           ng-if="$ctrl.tab === 'newsletter'">
+        <div class="form-group">
+          <label>
+            <input type="checkbox"
+                   ng-disabled="!$ctrl.hasNewsletter"
+                   ng-checked="$ctrl.showNewsletterForm"
+                   ng-model="$ctrl.showNewsletterForm"/> Display Newsletter Form?
+          </label>
+        </div>
+        <div class="form-group">
+          <h6 ng-if="$ctrl.hasNewsletter">This form will allow donors to subscribe to your MPDX newsletter.</h6>
+          <h6 ng-if="!$ctrl.hasNewsletter">This option is disabled because you do not have an MPDX account. To sign up for one, visit <a href="mpdx.org" target="_blank">mpdx.org</a>.</h6>
```

Comment:
Should this `href` have the protocol on it?

Reply:
doesn't matter. I'll add https though.